packages/runtime/src/store/index.ts

```diff
@@ -353,3 +365,7 @@ export class TutorialStore {
     this._themeRef.set(this._themeRef.get() + 1);
   }
 }
+
+function filterEntries<T extends object>(obj: T, filter: string[]) {
+  return Object.fromEntries(Object.entries(obj).filter(([entry]) => filter.includes(entry)));
+}
```

Comment:
That seems a bit overkill, I would use a much simpler function for what we're doing here because we already know that each value in `filter` is expected to be an entry in `obj`. 

Let's rename this function `pick` and do:

```ts
function pick<T>(obj: Record<string, T>, entries: string[]): Record<string, T> {
  const result = {};

  for (const entry of entries) {
    // kept to avoid having undefined values, but really if this is hit this is a
    // bug so maybe we should log something if we have an entry that is not present
    if (entry in obj) {
      result[entry] = obj[entry];
    }
  }

  return result;
}
```

Reply:
Looks better,  nice!